scdorr86/FE-Capstone-cd
# Additional stretch

## Description
imported a logo. 
comments can now be deleted by commenting user
when a post is deleted all associated comments are deleted from DB as well
rsvp page refreshes upon comment delete
form input resets on comment add

## Related Issue
adds comment functionality. fixes refresh bug as well as form reset bug.

## Motivation and Context
improves functionality. another layer of CRUD accomplished

## How Can This Be Tested?
comments were added and deleted to test refresh and form reset. post was created and comments were added to it then it was deleted to see if it's associated comments were deleted as well.

## Screenshots (if appropriate):

## Types of changes
- [x ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
